silverback/types.py

```diff
@@ -29,6 +34,31 @@ class SilverbackStartupState(BaseModel):
     last_block_processed: int
 
 
+class CronJob(BaseModel):
+    last_run: datetime = datetime.utcnow()
+    schedule: str
+
+    @property
```

Comment:
It looks to me like these can be `functools.cached_property`. I don't think these should be mutable, so writing these via cached_property might make more sense.

Reply:
Don't think we gain much there, but can do.